### PR TITLE
docs: Update README for emissary and KFP 1.7. Partial #6306

### PR DESCRIPTION
**Pull request by zijianjoy**
**Description of your changes:**

Partial #6306

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,9 +13,13 @@ The Kubeflow pipelines service has the following goals:
 * Easy experimentation: making it easy for you to try numerous ideas and techniques, and manage your various trials/experiments.
 * Easy re-use: enabling you to re-use components and pipelines to quickly cobble together end to end solutions, without having to re-build each time.
 
-## Documentation
+## Installation
+
+* Install Kubeflow Pipelines from [an overview of several options](https://www.kubeflow.org/docs/pipelines/installation/overview/).
 
-Install Kubeflow Pipelines from [an overview of several options](https://www.kubeflow.org/docs/pipelines/installation/overview/).
+* :star: [Alpha] Starting from Kubeflow Pipelines 1.7, try out [Emissary Executor](https://www.kubeflow.org/docs/components/pipelines/installation/choose-executor/#emissary-executor) for not restricted by container runtime, and more secure pod access. (Docker runtime is no longer default option since Kubernetes 1.19)
```
Comment (Bobgy) [inline README.md:20]: For discussion, I might prefer terms like:

* :star: [Alpha] Starting from Kubeflow Pipelines 1.7, try out [Emissary Executor](https://www.kubeflow.org/docs/components/pipelines/installation/choose-executor/#emissary-executor). Emissary executor is container runtime agnostic. The default docker executor depends on docker container runtime, which will be deprecated on Kubernetes 1.20+.

Comment (zijianjoy) [reply README.md]: Sounds good, updated the document with suggested term.

File: README.md [hunk 2]
```diff
@@ -13,9 +13,13 @@ The Kubeflow pipelines service has the following goals:
 * Easy experimentation: making it easy for you to try numerous ideas and techniques, and manage your various trials/experiments.
 * Easy re-use: enabling you to re-use components and pipelines to quickly cobble together end to end solutions, without having to re-build each time.
 
-## Documentation
+## Installation
+
+* Install Kubeflow Pipelines from [an overview of several options](https://www.kubeflow.org/docs/pipelines/installation/overview/).
```
Comment (shuesc1) [inline README.md:18]: "...from choices described in [Installation Options for Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/installation/overview/)."

File: README.md [hunk 3]
```diff
@@ -13,9 +13,13 @@ The Kubeflow pipelines service has the following goals:
...
 
-Install Kubeflow Pipelines from [an overview of several options](https://www.kubeflow.org/docs/pipelines/installation/overview/).
+* :star: [Alpha] Starting from Kubeflow Pipelines 1.7, try out [Emissary Executor](https://www.kubeflow.org/docs/components/pipelines/installation/choose-executor/#emissary-executor). Emissary executor is container runtime agnostic. The default Docker executor depends on Docker container runtime, which will be deprecated on Kubernetes 1.20+.
```
Comment (shuesc1) [inline README.md:20]: "Emissary executor is container runtime agnostic meaning you are able to [do something -- describe benefit of this]."

File: README.md (hunk 2 above)
Comment (zijianjoy) [reply README.md:18]: Applied the suggestion.

File: README.md (hunk 3 above)
Comment (zijianjoy) [reply README.md:20]: Sounds good, added explanation with link to `Container runtimes`.

**Comment by Bobgy** [comment]
/lgtm
/approve
Thank you for the update!

**Pull request merged**